dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.1/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/CommonBindTest.java

```diff
@@ -98,24 +118,56 @@ public class CommonBindTest {
     protected static final String UNBOUNDID_USER_DN = "uid=" + UNBOUNDID_USER + "," + UNBOUNDID_BASE_DN;
     protected static final String UNBOUNDID_PWD = "usrpwd";
 
-    static int LDAP_PORT = ApacheDSandKDC.getLdapPort();
-    static int KDC_PORT = ApacheDSandKDC.getKdcPort();
+    static int LDAP_PORT = 389;//ApacheDSandKDC.getLdapPort();
+    static int KDC_PORT = 88;//ApacheDSandKDC.getKdcPort();
 
     protected static String[] stopStrings = null;
 
     @BeforeClass
     public static void setup() throws Exception {
-        ticketCacheFile = ApacheDSandKDC.getDefaultTicketCacheFile();
+        if ("true".equals(System.getProperty("fat.test.use.remote.docker"))) {
```

Comment:
fat.test.use.remote.docker is a flag to allow users to force the use of a remote docker host.
This property doesn't actually represent the use of a remote docker host at runtime.

I would recommend
```suggestion
        if (ExternalDockerClientFilter.instance().isValid()) {
```